js/styles.js

```diff
@@ -9,6 +9,16 @@ function categoryToName(category) {
   else { return "none"; }
 }
 
+function getColor(category){
```

Comment:
I think the categoryToColor function should take an optional second argument, which is the alpha value you want. Then we can use a single function.

Reply:
Good idea - and @ldecicco-USGS remember to use your IE-compatible pattern for optional arguments! (I'm learning from my mistake yesterday =) )